src/Error/BaseErrorHandler.php

```diff
@@ -15,7 +15,6 @@
 namespace Cake\Error;
 
 use Cake\Core\Configure;
-use Cake\Error\PHP7ErrorException;
```

Comment:
We should try and add a test so that the `PHP7ErrorException` usage is covered by code coverage.